Stripe/StripeError+Internal.m

```diff
@@ -0,0 +1,24 @@
+//
+//  NSError+StripeInternal.m
+//  Stripe
+//
+//  Created by Ben Guo on 6/22/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+#import "StripeError+Internal.h"
+#import "StripeError.h"
+
+@implementation NSError (StripeInternal)
+
++ (NSError *)stp_ephemeralKeyDecodingError {
+    NSDictionary *userInfo = @{
+                               NSLocalizedDescriptionKey: [self stp_unexpectedErrorMessage],
+                               STPErrorMessageKey: @"Failed to decode the ephemeral key. Make sure your backend is sending the unmodified JSON of the ephemeral key to your app."
+                               };
+    return [[self alloc] initWithDomain:StripeDomain code:STPEphemeralKeyDecodingError userInfo:userInfo];
+}
+
+@end
+
+void linkNSErrorInternalCategory(void) {}
```

Comment:
I think we can actually move this code into a single `StripeError.m` implementation but we would will still need the `StripeError+Internal.h`.

I did this recently with the `STPAPIClient (Sources) / STPAPIClient (SourcesPrivate)`:

https://github.com/stripe/stripe-ios/blob/master/Stripe/PublicHeaders/STPAPIClient.h#L238
https://github.com/stripe/stripe-ios/blob/master/Stripe/STPAPIClient%2BPrivate.h